base/ltfssaxes.dtx

```diff
@@ -192,503 +250,3177 @@
+\DeclareFontSeriesChangeRule {slsc}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {sl}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {slsx}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {slx}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {slex}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {slux}{bx} {bx}  {b}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbuc}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {sbec}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {sbc}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {sbsc}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {sb}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {sbsx}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {sbx}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {sbex}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {sbux}{bx} {bx}  {b}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {buc}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {bec}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {bc}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {bsc}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {b}{bx}    {bx}  {b}
+\DeclareFontSeriesChangeRule {bsx}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {bx}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {bex}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {bux}{bx}  {bx}  {b}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebuc}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ebec}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ebc}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {ebsc}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {eb}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {ebsx}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ebx}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {ebex}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ebux}{bx} {bx}  {b}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ubuc}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ubec}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ubc}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {ubsc}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ub}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {ubsx}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ubx}{bx}  {bx}  {b}
+\DeclareFontSeriesChangeRule {ubex}{bx} {bx}  {b}
+\DeclareFontSeriesChangeRule {ubux}{bx} {bx}  {b}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {uc}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {ec}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {c}{bx}    {bx}  {b}
+\DeclareFontSeriesChangeRule {sc}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {m}{bx}    {bx}  {b}
+\DeclareFontSeriesChangeRule {sx}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {x}{bx}    {bx}  {b}
+\DeclareFontSeriesChangeRule {ex}{bx}   {bx}  {b}
+\DeclareFontSeriesChangeRule {ux}{bx}   {bx}  {b}
+%    \end{macrocode}
+%
+%
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {m}{b}     {b}   {bx}
+%    \end{macrocode}
```

Comment:
yes I guess that makes sense.

Reply:
Is the same true for all virtual fonts which use CM/LM, though?

Reply:
Rethinking this. If we keep the rule, it doesn't hurt, because any "normal" font outside of the CM/LM world does implement 'b' and  so the third argument applies and if it doesn't implement 'b' and there is a 'bx' switching to that is not bad and if neither exist then we still end up with 'b' substituted to 'm' so we don't loose anything other than making one more attempt. And it might help in  any special fonts in the area of CM (as @cfr42 remarked).

Reply:
So I decided to re-add the rule.